v3/tasks/20250313-v3-quantamm/index.ts

```diff
@@ -0,0 +1,70 @@
+
+import * as expectEvent from '@helpers/expectEvent';
+
+import { saveContractDeploymentTransactionHash } from '@src';
+import { Task, TaskMode, TaskRunOptions } from '@src';
+import { QuantAMMDeploymentInputParams, createPoolParams } from './input';
+
+export default async (task: Task, { force, from }: TaskRunOptions = {}): Promise<void> => {
+  const input = task.input() as QuantAMMDeploymentInputParams;
+
+  const args = [input.Vault, input.PauseWindowDuration, input.FactoryVersion, input.PoolVersion];
+  const chainlinkEthOracleWrapper = await task.deployAndVerify('ChainlinkOracle', args, from, force);
+  const chainlinkBtcOracleWrapper = await task.deployAndVerify('ChainlinkOracle', args, from, force);
+  const chainlinkUsdcOracleWrapper = await task.deployAndVerify('ChainlinkOracle', args, from, force);
+  const updateWeightRunner = await task.deployAndVerify('UpdateWeightRunner', args, from, force);
```

Comment:
And then this would be:

```suggestion
  // Where is QuantAMMAdmin?
  const weightRunnerArgs = [input.ChainlinkDataFeedETH, input.QuantAMMAdmin];
  const updateWeightRunner = await task.deployAndVerify('UpdateWeightRunner', args, from, force);
```

Reply:
yea noticed this this morning, args are now updated for all contracts already